Makefile

```diff
@@ -79,10 +79,16 @@ deps: bin/golangci-lint bin/trivy bin/regcopy bin/jq bin/yq bin/crane ## Install
 tests-modules: ## Run unit tests for modules hooks and templates.
 	go test -timeout=${TESTS_TIMEOUT} -vet=off ./modules/... ./global-hooks/... ./ee/modules/... ./ee/fe/modules/...
 
+.ONESHELL:
 tests-matrix: ## Test how helm templates are rendered with different input values generated from values examples.
   ##~ Options: FOCUS=module-name
-	go test ./testing/matrix/ -v
-
+  # TODO: Need to find a proper way to maintain these symlinks.
```

Comment:
Thinking about it a little bit more.
1. Maybe it is better to move setup and teradown to the golang tests.
2. How do we test OpenStack and vSphere cloud providers?  I assume there are checksum helpers in these folders.

Reply:
1. Moved work with symlinks to matrix tests code.
2. This case is taken into consideration.